examples/typescript/example.ts

```diff
@@ -35,8 +36,56 @@ vm.attachChangeEvent(() => {
     console.log("Something changed!");
 });
 
+// VB-Audio Callback example:
+
+// Index at which the first bus output starts in inputChannels
+let inputOffset = 0;
+switch (vm.$type) {
+    case "voicemeeter": {
+        inputOffset = 12;
+        break;
+    }
+    case "voicemeeterBanana": {
+        inputOffset = 22;
+        break;
+    }
+    default: {
+        inputOffset = 34;
+        break;
+    }
+}
+// This callback just passes through the audio signal to the output channels, without any changes
+const simplePassthroughCallback = (arg: types.AudioCallbackArg) => {
+    if (arg.command === AudioCallbackCommands.BUFFER_MAIN) {
+        const { outputChannels, inputChannels, samplesPerFrame } = arg.data;
+        for (const [ch, outputChannel] of outputChannels.entries()) {
+            console.log(outputChannel[0]);
+            for (let i = 0; i < samplesPerFrame; i++) {
+                outputChannel[i] = inputChannels[inputOffset + ch][i];
+            }
+        }
+    }
+    if (arg.command === AudioCallbackCommands.STARTING) {
+        console.log(`Started audio callback with sampleRate: ${arg.data.sampleRate}`);
+    }
+    if (arg.command === AudioCallbackCommands.ENDING) {
+        console.log("Stopped audio callback");
+    }
+};
+// Register the audio callback
+vm.registerAudioCallback(AudioCallbackModes.MAIN, simplePassthroughCallback, "ConnectorApp");
+
+// Start the audio callback
+vm.startAudioCallback();
+
+// Stop the audio callback after 5s
+setTimeout(() => {
+    console.log("Unregistering audio callback");
+    vm.unregisterAudioCallback(AudioCallbackModes.MAIN);
+}, 5000);
+

```

Comment:
Is this example too complex? Should the example be changed to just log callback start/end and leave it at that?

Reply:
I think it is okay. Maybe we could think about creating multiple examples, but i have no hard feelings on this.

Reply:
What do you think about maybe a separate example file for audio callbacks? So the basic example file remains a bit simpler?

Reply:
yep. Lets create an additional example for this 👍🏼